oasislmf/model_preparation/gul_inputs.py

```diff
@@ -284,7 +284,7 @@ def get_gul_input_items(
             ['peril_id', 'coverage_type_id', 'areaperil_id', 'vulnerability_id'] +
             (['model_data'] if 'model_data' in gul_inputs_df else []) +
             ([SOURCE_IDX['loc']] if SOURCE_IDX['loc'] in gul_inputs_df else []) +
-            ['is_bi_coverage', 'group_id', 'item_id', 'coverage_id', 'layer_id', 'agg_id', 'summary_id', 'summaryset_id']
+            ['is_bi_coverage', 'group_id', 'item_id', 'coverage_id', 'layer_id', 'agg_id', 'summary_id', 'summaryset_id', 'status']
```

Comment:
The `summary_id` and `summaryset_id` columns can be removed also because they're used to write the GUL summary xref files, which has now been moved into the summaries module. But it's OK, I have made this change in my current FM-related feature branch, which will end up in `develop`.